src/odemis/driver/smaract.py

```diff
@@ -4218,20 +4225,29 @@ def __init__(self, name, role, locator, channels, ref_on_init=False, precision_m
         """
         model.HwComponent.__init__(self, name, role, *args, **kwargs)
 
+        if not smaract_python_sdk:
```

Comment:
I can't comment on the lines above, but I think it might be a good idea to mention that connecting using the serial number has caused problems.

``` 
            The device can also be addressed by its serial number. Note: this has caused connection issue on some of the devices.
                usb:sn:<serial_number>
                network:sn:<serial_number>
```

Reply:
We are in contact with smaract and it seems the issue can be resolved